authors/apps/authentication/models.py

```diff
@@ -8,11 +8,12 @@
 )
 from django.db import models
 
+
```

Comment:
Please refrain from including unnecessary changes such as this, line 16, 37, 38 and any others unrelated to your PT story.

Reply:
For the unnecessary changes, this came as a result of reformatting the document to pep8.

Reply:
Going forward, ensure that the formatting does not introduce changes such as these. I recommend using pylint instead and not relying on autoformatting.